docs/ref/django-admin.txt

```diff
@@ -1372,9 +1372,10 @@ If only the project name is given, both the project directory and project
 package will be named ``<projectname>`` and the project directory
 will be created in the current working directory.
 
-If the optional destination is provided, Django will use that existing
-directory as the project directory, and create ``manage.py`` and the project
-package within it. Use '.' to denote the current working directory.
+If the optional destination is provided, Django will use that name as the
+project directory, and create ``manage.py`` and the project package within it.
+If the directory with the given name doesn't exist, it will be created. Use '.'
+to denote the current working directory.
 
 For example:
```

Comment:
Can we call out in the example what this new feature is doing? Something like

For example:

    django-admin startapp myapp /Users/jezdez/Code/myapp/newdirectory

Will create a new directory called `newdirectory` where the app will have its code placed.

Reply:
I'm not sure this is a good example... The example shopws the `startproject` command, and as written currently, it implies that the project is being created at `/Users/jezdez/Code/myproject_repo`. 

I could however add a line below the command:

> It will create the directory `myproject_repo` if needed, where the project will have its code placed.

What do you think?